src/parser/jobs/mnk/changelog.tsx

```diff
@@ -2,13 +2,18 @@ import CONTRIBUTORS from 'data/CONTRIBUTORS'
 import React from 'react'
 
 export const changelog = [
+	{
+		date: new Date('2024-02-19'),
+		Changes: () => <>Improve Beast Chakra gauge visualisation.</>,
+		contributors: [CONTRIBUTORS.AKAIRYU],
+	},
 	{
 		date: new Date('2023-10-24'),
 		Changes: () => <>Fix a bug with the calculation for the expected number of Masterful Blitz uses in Riddle of Fire.</>,
 		contributors: [CONTRIBUTORS.AKAIRYU],
 	},
 	{
-		date: new Date('2023-10-23'),
+		date: new Date('2023-07-09'),
 		Changes: () => <>Add gauge visualisation for Perfect Balance, and mark as supported for Endwalker.</>,
 		contributors: [CONTRIBUTORS.AKAIRYU],
 	},
```

Comment:
Is this a merge conflict problem or something causing the date change on this old changelog entry?  This should be unchanged, correct?

Reply:
Yeah, just a goof. Should be fixed now.